cpp/scenario/gazebo/src/GazeboSimulator.cpp

```diff
@@ -125,7 +137,6 @@ class GazeboSimulator::Impl
 // ===============
 // GazeboSimulator
 // ===============
-
```

Comment:
Sure. I actually didn't check for any code-style convention and simply used the default formatter that comes with VSCode. Is there a preferred one?

Reply:
There's a `.clang-format` that your IDE can use. I'm not sure if VSCode detects it automatically, but AFAIK the configuration should not remove blank lines, unless they are more than 1.

Reply:
Hm, it seems to be picked up by VSCode. It could be that I put a function there for debugging purposes and didn't notice that the space went missing once I cleaned it up. I've restored it.